packages/sdk-core/src/model/http/SubmissionUrlInformation.ts

```diff
@@ -90,22 +99,33 @@ export class SubmissionUrlInformation {
 
     public static changeSubmissionFormat(submissionUrl: string, desiredFormat: 'json' | 'plcrash' | 'minidump') {
         const submitIndex = submissionUrl.indexOf(this.SUBMIT_PREFIX);
-        const url = new URL(submissionUrl);
+
         if (submitIndex !== -1) {
-            const pathParts = url.pathname.split('/');
+            const queryParametersIndex = submissionUrl.indexOf('?');
+            const queryParameters = queryParametersIndex === -1 ? '' : submissionUrl.substring(queryParametersIndex);
+            const pathname = submissionUrl.substring(
+                submitIndex + this.SUBMIT_PREFIX.length,
+                queryParametersIndex === -1 ? undefined : queryParametersIndex,
+            );
+            const pathParts = pathname.split('/');
             // path parts are prefixed with '/' character. Expected and valid submit format is:
             // /universe/token/format
             // splitting pathname should generate at least 4 elements ('', universe, token, format)
             // if pathParts length is not equal to 4 then the invalid were passed.
-            const expectedMinimalPathParts = 4;
+            const expectedMinimalPathParts = 3;
             if (pathParts.length < expectedMinimalPathParts) {
                 return submissionUrl;
             }
-            pathParts[3] = desiredFormat;
-            url.pathname = pathParts.join('/');
+            pathParts[2] = desiredFormat;
+            return (
+                submissionUrl.substring(0, submitIndex + this.SUBMIT_PREFIX.length) +
+                pathParts.join('/') +
+                queryParameters
+            );
         } else {
+            const url = new URL(submissionUrl);
```

Comment:
You still use `URL` here.

Reply:
As I said - we're using only supported URL API. If something is unsupported I'm creating the URL manually.